packages/reports/src/Containers/statement.tsx

```diff
@@ -87,22 +87,28 @@ const getRowAction: TGetRowAction = (row_obj: TSource | TRow) => {
     let action: TAction = {};
     if (row_obj.id && ['buy', 'sell'].includes(row_obj.action_type)) {
         const contract_type = extractInfoFromShortcode(row_obj.shortcode)?.category?.toUpperCase();
-        action =
-            getSupportedContracts()[contract_type as TSupportedContractType] &&
-            !isForwardStarting(row_obj.shortcode, row_obj.purchase_time || row_obj.transaction_time)
-                ? getContractPath(row_obj.id)
-                : {
-                      message: '',
-                      component: (
-                          <Localize
-                              i18n_default_text="The {{trade_type_name}} contract details aren't currently available. We're working on making them available soon."
-                              values={{
-                                  trade_type_name:
-                                      getUnsupportedContracts()[contract_type as TUnsupportedContractType]?.name,
-                              }}
-                          />
-                      ),
-                  };
+        const unsupported_contract_config = getUnsupportedContracts()[contract_type as TUnsupportedContractType];
+        action = unsupported_contract_config
+            ? {
+                  message: '',
+                  component: (
+                      <Localize
+                          i18n_default_text="The {{trade_type_name}} contract details aren't currently available. We're working on making them available soon."
+                          values={{
+                              trade_type_name: unsupported_contract_config?.name,
+                          }}
+                      />
+                  ),
+              }
+            : getContractPath(row_obj.id);
+        if (
+            isForwardStarting(row_obj.shortcode, row_obj.purchase_time || row_obj.transaction_time) &&
+            !hasForwardContractStarted(row_obj.shortcode)
```

Comment:
He we need to check if it's forward starting contract, because extracted from `shortcode` `start_time` contains letter 'F' in the end and inside of `hasForwardContractStarted()` we crop it in order to get real `start_time`. If it will be NOT forward staring contract and we crop the `start_time`, we remove necessary numbers, not letter 'F"